src/main/g8/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+.idea/
```

Comment:
The .idea folder should be excluded by a user's global gitignore file. I recommend to only put build tool related entires into the project's gitignore.

Reply:
Agreed, I do the same with my own projects. But the goal here is to lower the barrier for newcomers.